.github/workflows/Build_Docker_Image_PR.yml

```diff
@@ -16,3 +16,16 @@ jobs:
           persist-credentials: false
       - name: Build image
         run: docker build -t test-build .
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@369eb591f429131d6889c46b94e711f089e6ca96 # tag=v5.6.1
+        with:
+          images: test-build
+      - name: Build and push Docker image
+        id: push
+        uses: docker/build-push-action@ca877d9245402d1537745e0e356eab47c3520991 # tag=v6.13.0
```

Comment:
question: I'm not sure I get the goal of those two actions. We are already building a test image at L17, so why are we building again at L24 with push=false ?

Reply:
Because I forgot to remove the initial `docker build` ^^

Reply:
haha ok